studio/src/components/List/QuickEdit.js

```diff
@@ -95,6 +95,10 @@ function QuickEdit({ data, setID, slug, page = false, onQuickEditUpdate = () =>
       <Form.Item name="published_date" label="Published Date">
         <DatePicker />
       </Form.Item>
+      <Form.Item label="Created Date">
+        <span>{createdAt ? dayjs(createdAt).format('YYYY-MM-DD') : 'N/A'}</span>
```

Comment:
@pallavimshr please use [Typography](https://ant.design/components/typography) component for this instead of a `span` tag.